cakechat/api/utils.py

```diff
@@ -7,8 +8,8 @@ def get_api_error_response(message, code, logger):
 
 
 def _is_list_of_unicode_strings(data):
-    return (isinstance(data, list) or isinstance(data, tuple)) and len(data) > 0 \
-           and all(isinstance(s, unicode) for s in data)
+    return (data and isinstance(data, (list, tuple)) and
```

Comment:
if you pass an empty list of None to this function it would return `[]` and `None` respectively
probably it's better to cast the result to bool explicitly